### ISSUE TITLE
GRIPSS error configuring filter thresholds

### ISSUE THREAD

**Issue by lkhilton** [hartwigmedical/hmftools#110]
I'm running GRIPSS and trying to specify alternate values to the `--hard_max_normal_absolute_support`, `-hard_max_normal_relative_support`, and `-soft_max_normal_relative_support` values at the command line. If I include any of these arguments in my command, I get an error: 

```
Exception in thread "main" java.lang.NumberFormatException: null
        at java.base/java.lang.Integer.parseInt(Integer.java:614)
        at java.base/java.lang.Integer.parseInt(Integer.java:770)
        at com.hartwig.hmftools.common.cli.Configs.defaultIntValue(Configs.java:64)
        at com.hartwig.hmftools.gripss.GripssFilterConfig$Companion.createConfig(GripssConfig.kt:156)
        at com.hartwig.hmftools.gripss.GripssConfig$Companion.createConfig(GripssConfig.kt:68)
        at com.hartwig.hmftools.gripss.GripssApplicationKt.main(GripssApplication.kt:40)
```

If I leave these args out of my command, GRIPSS launches normally. 

I'm running GRIPSS 1.2 installed via the `hmftools-gripss` bioconda recipe.

**Comment by lkhilton** [comment]
For reference, this is the command I ran: 

```
gripss -Xms4G -Xmx16G \
-ref_genome references/grch37/genome_fa/genome.fa \
-breakend_pon hmftools-references/gridss/gridss_pon_single_breakend.hg19.bed \
-breakpoint_pon hmftools-references/gridss/gridss_pon_breakpoint.hg19.bedpe  \
-breakpoint_hotspot hmftools-references/gridss/KnownFusionPairs.hg19.bedpe \
-input_vcf gridss_viral_annotation.vcf.gz \
-output_vcf gridss_somatic.vcf.gz \
-tumor 05-10981T \
-reference 05-10981N \
-hard_max_normal_absolute_support 5
```

**Comment by jonbaber** [comment]
Thanks for letting me know. I can reproduce the problem. 

Will have a fix very shortly.

**Comment by jonbaber** [comment]
Fixed in [1.4](https://github.com/hartwigmedical/hmftools/releases/tag/gripss-v1.4)
